docs/source/release/v4.0.0/diffraction.rst

```diff
@@ -62,57 +57,54 @@ Bugfixes
 - :ref:`SaveIsawPeaks <algm-SaveIsawPeaks>` does not have duplicate peak numbers when saving PeaksWorkspaces with more than one RunNumber.
 - :ref:`LoadIsawPeaks <algm-LoadIsawPeaks>` now loads the calibration from the peaks file correctly.
-
 - :ref:`OptimizeCrystalPlacement <algm-OptimizeCrystalPlacement>` now updates the sample location used by peaks.  Previously, the sample was effectively left unmoved. Default for indexing tolerance was lowered to 0.15 and can now be called more than once without error.
 
 Powder Diffraction
 ------------------
 
+New Algorithms
+##############
+
+- :ref:`HB2AReduce <algm-HB2AReduce>` algorithm reduces HFIR POWDER (HB-2A) data.
+- :ref:`LoadGudrunOutput <algm-LoadGudrunOutput>` is a new algorithm that allows users to load the standard Gudrun output files into Mantid.
+- :ref:`PDConvertReciprocalSpace <algm-PDConvertReciprocalSpace>` new algorithm to convert between reciprocal space units.
+- :ref:`PDConvertRealSpace <algm-PDConvertRealSpace>` new algorithm to convert between real space units.
+
 Improvements
 ############
 
 - Focusing in texture mode for Gem now properly saves .gda files.
-- Focusing on Gem now crops values that would be divided by very small or zero vanadium values
+- Focusing on Gem now crops values that would be divided by very small or zero vanadium values.
 - Removed save_angles flag for Gem , as it was set by the texture mode.
 - Added save_all flag to Gem that is set to true by default, setting it to false disables the saving of .NXS files.
-- Added subtract_empty_instrument flag to Gem that is true by default, setting it to false disables subrtracting the empty.
+- Added subtract_empty_instrument flag to Gem that is true by default, setting it to false disables subtracting the empty.
 - Changed spline coefficient so that the default for long_mode on and long_mode off can be set separately.
 - Focus on Pearl now has a focused_bin_widths parameter in pearl_advanced_config.py to allow setting default rebin values.
 - Focus on Pearl now saves out xye_tof files.
 - :ref:`PDLoadCharacterizations <algm-PDLoadCharacterizations>` now sets the same run numbers for all rows when using an ``exp.ini`` file.
 - Focus now checks if the vanadium for a run is already loaded before loading it in to prevent reloading the same vanadium multiple times.
 - :ref:`SaveReflections <algm-SaveReflections>` now supports saving indexed modulated peaks in the Jana format.
-- `PyStoG <https://pystog.readthedocs.io/en/latest/>`_ has been added as an external project
+- `PyStoG <https://pystog.readthedocs.io/en/latest/>`_ has been added as an external project.
 
 Bugfixes
 ########
 
-- multiple_scattering flag is now optional for Polaris focus when absorb_correction is true.
+- Multiple_scattering flag is now optional for Polaris focus when absorb_correction is true.
```

Comment:
`multiple_scattering` may be better as unformatted text (and with correct captialisation) if it is something in code (which this suggests it is).